src/libs/actions/Session/index.js

```diff
@@ -170,33 +170,11 @@ function resendValidateCode(login = credentials.login) {
             onyxMethod: Onyx.METHOD.MERGE,
             key: ONYXKEYS.ACCOUNT,
             value: {
-                isLoading: true,
                 errors: null,
-                loadingForm: CONST.FORMS.VALIDATE_CODE_FORM,
-            },
-        },
-    ];
-    const successData = [
-        {
-            onyxMethod: Onyx.METHOD.MERGE,
-            key: ONYXKEYS.ACCOUNT,
-            value: {
-                isLoading: false,
-                loadingForm: null,
-            },
-        },
-    ];
-    const failureData = [
-        {
-            onyxMethod: Onyx.METHOD.MERGE,
-            key: ONYXKEYS.ACCOUNT,
-            value: {
-                isLoading: false,
-                loadingForm: null,
             },
```

Comment:
What's the reasoning for removing these entirely versus copying the changes to each?